tools/dist/index.js

```diff
@@ -21,17 +22,28 @@ const exec = async (file, args, options = {}) => {
   return execa(file, args, opts);
 };
 
+const rmDir = dir =>
+  new Promise((resolve, reject) => {
+    const fullPath = path.resolve(__dirname, rootFolder, dir);
```

Comment:
can you directly inline `../..` here ?

Reply:
I wanted to make the relative path to the root folder more obvious, just in case the file was moved (like it happened before), so I moved it to the top of the file, but I can inline it back.